lib/service/Service.ts

```diff
@@ -133,72 +76,269 @@ class Service extends EventEmitter {
   /**
    * Get the list of the order book's available pairs.
    * @returns A list of available trading pairs
+   * @throws ServiceError with status.FAILED_PRECONDITION code
    */
   public getPairs = () => {
-    return this.orderBook.getPairs();
+    try {
+      return this.orderBook.getPairs();
+    } catch (err) {
+      const serviceError: ServiceError = {
+        code: status.FAILED_PRECONDITION,
+        metadata: err,
+        name: 'db',
+        message: 'could not get pairs',
+      };
+      throw serviceError;
+    }
   }
 
   /**
    * Get a list of standing orders from the order book.
+   * @throws ServiceError with status.INTERNAL code
    */
   public getOrders = ({ pairId, maxResults }: { pairId: string, maxResults: number }) => {
-    return this.orderBook.getPeerOrders(pairId, maxResults);
+    try {
+      return this.orderBook.getPeerOrders(pairId, maxResults);
+    }catch (err) {
+      const serviceError: ServiceError = {
+        code: status.INTERNAL,
+        metadata: err,
+        name: 'internal',
+        message: 'could not get orders',
+      };
+      throw serviceError;
+    }
   }
 
   /**
    * Add an order to the order book.
+   * @throws ServiceError with status.INTERNAL code
```

Comment:
`@throws` is not a keyword for typedoc, same for other lines where @throws is used: http://typedoc.org/guides/doccomments/

I might just drop these @throws comment lines entirely, if there's something important or unique to note maybe put it in the method description.

Reply:
sure, will fix